lib/topic.ts

```diff
@@ -7,6 +7,7 @@ export const navigateToKafkaTopicsList = async function (page: Page, kafkaName:
   await page.locator('a', { hasText: kafkaName }).click();
   // data-testid=pageKafka-tabTopics
   await page.locator('button', { hasText: 'Topics' }).click();
+  await page.getByText('Topics').click();
```

Comment:
Isn't this duplicating the line above? I'm not aware of any secondary topics button that you would need to press after opening the kafka dashboard.

Reply:
Replaced both lines for a more robust option to click on the 'Topics' Tab:

`await page.getByTestId('pageKafka-tabTopics').click();`

Reply:
Can I resolve this conversation? @mmusil